components/user_settings/security/user_settings_security.jsx

```diff
@@ -92,21 +92,18 @@ export default class SecurityTab extends React.Component {
         };
     }
 
-    componentDidMount() {
+    async componentDidMount() {
```

Comment:
maybe use this function to call a second async function that will also set the state

Reply:
is there reason to avoid async-lifecycle method?

Reply:
I'm not fully against it, but it should not be the case. you could go about using Fabric and add a suspend while you get the info that you need if you want to try that.

Reply:
I see the point. to avoid async logic directly into the lifecycle because we should extract it from components later (to integration suspense, react-cache, etc). so yeah, makes sense to me. :+1: